bigtable-client-core-parent/bigtable-hbase/src/main/java/com/google/cloud/bigtable/hbase/wrappers/veneer/BigtableHBaseVeneerSettings.java

```diff
@@ -219,6 +219,22 @@ public long getBatchingMaxRequestSize() {
     return batchingMaxMemory;
   }
 
+  @Override
+  public String getUserAgent() {
```

Comment:
Maybe getEffectiveUserAgent()?
Also if this exists just for CheckConfig, I think we should probably drop CheckConfig and keep this private

Reply:
Reverted the change and removed CheckConfig